code/Populate.php

```diff
@@ -102,60 +99,78 @@ public static function requireRecords($force = false): bool
         return true;
     }
 
-    /*
+    /**
      * Delete all the associated tables for a class
+     *
+     * @param string $className
      */
-    private static function deleteClass(string $class): void
+    private static function truncateObject(string $className): void
```

Comment:
While renaming the method, could it be rather named `truncateObjectTables`? The name of the update hook will play with it nicely, too.

Reply:
Yeah, great question. I thought a lot about this, and every time I play with naming around "tables", all the arguments feel like they don't match the context.

The interpretation I'm seeing here is: The user specifies an object, and the logic determines how to truncate that object. By changing the context to be about "tables" it feels like the user should be defining what tables should be truncated, which is a possible alternate design; but feels less useful to me. What do you think?